kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/steps/KafkaSteps.java

```diff
@@ -38,6 +42,29 @@ public static void createTopic(String clusterName, String topicName, String name
                 KafkaTopicTemplates.defaultTopic(namespace, clusterName, topicName, partitions, replicas, minIsr).build());
     }
 
+    /**
+     * Create topic using test clients.
+     *
+     * @param topicName the topic name
+     * @param bootstrap the bootstrap
+     * @param partitions the partitions
+     * @param replicas the replicas
+     */
+    public static void createTopicTestClient(String deployNamespace, String topicName, String bootstrap, int partitions, int replicas) {
+        String podName = KafkaUtils.AdminTestClient(deployNamespace, bootstrap);
+        String command = "admin-client topic create --bootstrap-server=" + bootstrap + " --topic=" + topicName + " --topic-partitions=" + partitions +
+                " --topic-rep-factor=" + replicas;
+
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        kubeClient().getClient().pods().inNamespace(deployNamespace).withName(podName)
+                .writingOutput(baos)
+                .writingError(baos)
+                .withTTY()
+                .usingListener(new KubeSimpleListener())
+                .exec("sh", "-c", command)
```

Comment:
Not quite sure why you run using a shell.

Reply:
I need to run the `admin-client` command inside the admin client pod, so I have used the `exec` function to do that. I have not found another way to do that.

Reply:
Why do we need to run inside the admin client pod? What can't we just connect using a java admin client here?

Reply:
Because the ports are not exposed for kafka, so we cannot connect from outside that easy AFAIK.

Reply:
You didn't address "Not quite sure why you run using a shell"

Reply:
For me was simpler to create a command variable writing everything in a line than separated by comma, that's the simple reason I did it in that way TBH. I didn't think about if it is better or not running in a shell